ocs_ci/ocs/platform_nodes.py

```
@@ -143,7 +143,11 @@ def get_nodes_platform(self):
         if config.ENV_DATA["platform"] == constants.BAREMETAL_PLATFORM:
             if config.ENV_DATA["deployment_type"] == "ai":
                 platform += "_ai"
-        if config.ENV_DATA["platform"] == constants.IBM_HCI_PLATFORM:
+        if (
+            config.ENV_DATA["platform"] == constants.IBM_HCI_PLATFORM
+            and not get_client_type_by_name(cluster_name)
+            == constants.HOSTED_CLUSTER_KUBEVIRT
+        ):
             if config.ENV_DATA["deployment_type"] == "ipi":
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)